codalab/model/bundle_model.py

```diff
@@ -763,6 +806,45 @@ def save_bundle(self, bundle):
                 self.do_multirow_insert(connection, cl_bundle_metadata, metadata_values)
                 bundle.id = result.lastrowid
 
+    def save_bundle_rest(self, bundle_info):
+        """
+        Save a bundle. On success, sets the bundle's id from the result.
+        Bundle info should be validated prior to calling this method.
+        """
+        dependency_values = bundle_info.pop('dependencies')
+        metadata_values = bundle_info.pop('metadata')
+
+        # Check to see if bundle is already present, as in a local 'cl cp'
+        if not self.batch_get_bundles(uuid=bundle_info['uuid']):
+            with self.engine.begin() as connection:
+                connection.execute(cl_bundle.insert().values(bundle_info))
+                self.do_multirow_insert(connection, cl_bundle_dependency, dependency_values)
+                self.do_multirow_insert(connection, cl_bundle_metadata, metadata_values)
+
+    def update_bundle_rest(self, bundle_update):
```

Comment:
I don't understand the plan here. Are you planning on deprecating the Bundle ORM objects? That seems like a big change. There's a ton of code that in the bundle_model and bundle_manager classes that uses the bundle ORM objects. 

 If not, then at what point will the update_bundle method below go away? Are you just planning on making copies of all the methods you need? That seems bad.